pkg/services/ngalert/schedule/schedule.go

```diff
@@ -373,7 +375,7 @@ func (sch *schedule) ruleEvaluationLoop(ctx context.Context) error {
 
 				itemFrequency := item.IntervalSeconds / int64(sch.baseInterval.Seconds())
 				if item.IntervalSeconds != 0 && tickNum%itemFrequency == 0 {
-					readyToRun = append(readyToRun, readyToRunItem{key: key, ruleInfo: ruleInfo})
+					readyToRun = append(readyToRun, readyToRunItem{key: key, ruleInfo: ruleInfo, version: itemVersion})
```

Comment:
I'm not sure I understand this change when the version is also in `ruleInfo`?

Reply:
I moved the version from the `alertRuleInfo` to `readyToRunItem` because the control structure is not copied any more.